docs/analysis-workflows/harmonization-dna-pipeline.md

```diff
@@ -0,0 +1,37 @@
+---
+id: dna-pipeline
+title: DNA-Seq Analysis Pipeline
+---
+
+The DNA-Seq analysis pipeline identifies various somatic variant types within Whole Exome Sequencing (WXS) and Whole Genome Squencing (WGS) data. The pipeline starts with a reference alignment step followed by co-cleaning to increase the alignment quality. DNA-Seq analysis is implemented across two main procedures:
+
+- Sequence Alignment
+- Variant Calling
+
+In the future, these procedures will be extended to include:
+
+- Variant Masking
+- Variant Annotation
+- Concencus Calling
+
+## Alignment
+
+Before genomes can be compared for variant analysis, they must be aligned to to a reference genome. In addition to alignment, the alignemtn step includes some data cleanup operations to correct for technical biases and make the data suitable for analysis.
+
+The alignemnt step uses:
+
+- **[BWA-MEM version 0.7.17-r1188](http://bio-bwa.sourceforge.net/)** as its main aligner and
+- **[Biobambam versino xxx](https://www.sanger.ac.uk/science/tools/biobambam)** to mark duplicates.
+- **[GrCh37 HL7](need link)** as the reference genome

```

Comment:
I believe we're using GrCh38 as the reference genome

Reply:
yeah and I googled how to write it - seemed to have the "R" capitalized https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26/